test/system/src/test/java/io/pravega/test/system/MultiControllerTest.java

```diff
@@ -156,32 +156,41 @@ public void multiControllerTest() throws ExecutionException, InterruptedExceptio
 
     private void withControllerURIDirect() throws ExecutionException, InterruptedException {
         Assert.assertTrue(createScopeWithSimpleRetry(
-                "scope" + RandomStringUtils.randomAlphanumeric(10), controllerURIDirect.get()).get());
+                "scope" + RandomStringUtils.randomAlphanumeric(10), controllerURIDirect.get()));
     }
 
     private void withControllerURIDiscover() throws ExecutionException, InterruptedException {
         if (!DOCKER_BASED) {
             Assert.assertTrue(createScopeWithSimpleRetry(
-                    "scope" + RandomStringUtils.randomAlphanumeric(10), controllerURIDiscover.get()).get());
+                    "scope" + RandomStringUtils.randomAlphanumeric(10), controllerURIDiscover.get()));
         }
 
     }
 
-    CompletableFuture<Boolean> createScopeWithSimpleRetry(String scopeName, URI controllerURI) {
+    private boolean createScopeWithSimpleRetry(String scopeName, URI controllerURI) throws ExecutionException, InterruptedException {
         // Need to retry since there is a delay for the mesos DNS name to resolve correctly.
-        return Retry.withExpBackoff(500, 2, 10, 5000)
+        @Cleanup
+        final ControllerImpl controllerClient = new ControllerImpl(ControllerImplConfig.builder()
```

Comment:
This client creation seems redundant, this method can invoke createScope() directly.
runInExecutor could be used instead of `runAsync()`

Reply:
@Prabha-Veerubhotla please ignore my previous comment in case you have intentionally created two clients as part of the test.